tests/test_geospatial.py

```diff
@@ -289,6 +289,53 @@ def test_raster_ingest(self):
                 self.assertIsInstance(fltr, tiledb.ZstdFilter)
                 self.assertEqual(fltr.level, 7)
 
+    def test_rgb_raster_ingest(self):
```

Comment:
This test only asserts that ingest doesn't fail and that the output has the same shape as the input. That goes a long way, though. Rasterio already does a thorough job of testing raster merger.

Reply:
I've visually checked the result and it's correct.